pkg/bindings/play/play.go

```diff
@@ -3,95 +3,23 @@ package play
 import (
 	"context"
 	"io"
-	"net/http"
-	"os"
-	"strconv"
 
-	"github.com/containers/image/v5/types"
-	"github.com/containers/podman/v4/pkg/auth"
-	"github.com/containers/podman/v4/pkg/bindings"
+	"github.com/containers/podman/v4/pkg/bindings/kube"
 	"github.com/containers/podman/v4/pkg/domain/entities"
-	"github.com/sirupsen/logrus"
 )
 
-func Kube(ctx context.Context, path string, options *KubeOptions) (*entities.PlayKubeReport, error) {
-	f, err := os.Open(path)
-	if err != nil {
-		return nil, err
-	}
-	defer f.Close()
-
-	return KubeWithBody(ctx, f, options)
+func Kube(ctx context.Context, path string, options *kube.PlayOptions) (*entities.PlayKubeReport, error) {
```

Comment:
This is a breaking change.

The signature of any current function must not change. Also types must not be touched. Type aliasing and wrapping around the new API by calling it here should resolve the issue.

Reply:
Same at other places.